src/services/news/docs.js

```diff
@@ -46,14 +46,13 @@ module.exports = {
 				target: {
 					type: 'string',
 					pattern: '[a-f0-9]{24}',
-					description: 'id of the scope the news belongs to.'
-					+ 'If no target is given, the news will belong to the school.',
+					description:
+						'id of the scope the news belongs to.' + 'If no target is given, the news will belong to the school.',
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)